NewRelic/NewRelicInteractor.php

```diff
@@ -32,11 +32,19 @@ public function setTransactionName($name)
     /**
      * {@inheritdoc}
      */
-    public function ignoreTransaction ()
+    public function ignoreTransaction()
     {
         newrelic_ignore_transaction();
     }
 
+    /**
+     * {@inheritdoc}
+     */
+    public function ignoreApdex()
```

Comment:
the method does not exists in the interface. Docblock is wrong, and if a project use the `LoggingInteractorDecorator` it wont work